.github/workflows/update-bidi-types.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow will update the WebdriverBidi types.
+
+name: Update WebdriverBidi types
+
+on:
+  schedule:
+    # Run hourly at xx:30.
+    - cron: '30 * * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build WebDriverBidi types
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
```

Comment:
I was initially thinking to use a submodule here, just like how we do with `wpt/`, for pinning purposes.

And then have the type generation be a pre-requisite step of e.g. `npm run build` (via `wireit`).

Have you considered this approach?

Reply:
I've considered this approach, but submoduling will low-key encourage modifying the spec locally which is something we don't want done. We can go either way, but we can settle with this solution for now and switch later if necessary.

Reply:
OK. I am fine with this solution for the time being but IMHO at first glance the submodule approach seems better. I could change my mind over time though, let's see.

Not intending to block this approach, but would like to see at least another supporter.

cc @OrKoN @Lightning00Blade